sdk/search/search-documents/README.md

```diff
@@ -25,30 +25,27 @@ Use the client library to:
 
 ## Getting started
 
-### Currently supported environments
-
-- [Node.js](https://nodejs.org/) version 8.x.x or higher
-
 ### Prerequisites
 
+- [Node.js](https://nodejs.org/) version 8.x.x or higher
 - An [Azure subscription][azure_sub].
 - An existing [Azure Cognitive Search][search_resource] resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
 
 If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-resource-name>` with your own unique names:
 
 ```PowerShell
-az search service create --resource-group <your-resource-group-name> --name <your-resource-name> --sku S
+az search service create --resource-group <your-resource-group-name> --name <your-resource-name> --sku Standard
 ```
 
 The above creates a resource with the "Standard" pricing tier. See [choosing a pricing tier](https://docs.microsoft.com/azure/search/search-sku-tier) for more information.
 
-### 1. Install the `@azure/search-documents` package
+- Install the `@azure/search-documents` package
```

Comment:
I don't think this looks right in the rich diff.  It should be "### Install the package" with one space separating the pound chars from the heading

Reply:
Done. Will be updated in the next commit